kitsune/bundles.py

```diff
@@ -124,21 +123,6 @@
         "source_filenames": ("sumo/js/questions.metrics-dashboard.js",),
         "output_filename": "build/questions.metrics-min.js",
     },
-    "questions.aaq.react": {
```

Comment:
We've still got a reference to this in kitsune/questions/jinja2/questions/new_question_react.html

This is rendered in kitsune/questions/views.py#aaq_react, which is rendered from #aaq if the new_aaq waffle flag is set.

I guess we want to tear all of that out too? Including the tests in kitsune/questions/tests/test_views.py:ReactAAQTests

Reply:
Do we also want a migration to remove the `new_aaq` flag from the db?

kitsune/questions/migrations/0004_new_aaq_waffle_flag.py is where its added

Reply:
I would go with no at this point. We can clean up any related artifacts after the release